hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcClient.java

```diff
@@ -875,7 +875,7 @@ public <T> T invokeSingle(final String nsId, RemoteMethod method,
    * @param method The remote method and parameters to invoke.
    * @param clazz Class for the return type.
    * @return The result of invoking the method.
-   * @throws IOException If the invoke generated an error.
+   * @throws IOException If to invoke generated an error.
```

Comment:
these the->to change isn't correct. Remove all such

Reply:
OK, I will fix it.